2025-04-16 00:17:55,177 - VocabMaster - INFO - 启动VocabMaster
2025-04-16 00:17:55,177 - VocabMaster - INFO - 使用GUI模式
2025-04-16 00:17:55,219 - VocabMaster - ERROR - 程序运行出错
Traceback (most recent call last):
  File "D:\MyProjects\VocabMaster\app.py", line 74, in main
    from gui import main
  File "D:\MyProjects\VocabMaster\gui.py", line 9, in <module>
    from utils import BECTest, TermsTest, DIYTest
  File "D:\MyProjects\VocabMaster\utils\__init__.py", line 4, in <module>
    from .base import TestBase
  File "D:\MyProjects\VocabMaster\utils\base.py", line 9, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
2025-04-16 00:17:55,220 - VocabMaster - ERROR - 未捕获的异常:
Traceback (most recent call last):
  File "D:\MyProjects\VocabMaster\app.py", line 81, in <module>
    main()
  File "D:\MyProjects\VocabMaster\app.py", line 74, in main
    from gui import main
  File "D:\MyProjects\VocabMaster\gui.py", line 9, in <module>
    from utils import BECTest, TermsTest, DIYTest
  File "D:\MyProjects\VocabMaster\utils\__init__.py", line 4, in <module>
    from .base import TestBase
  File "D:\MyProjects\VocabMaster\utils\base.py", line 9, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'

